pom.xml

```diff
@@ -102,6 +102,12 @@ SOFTWARE.
       <artifactId>javax.json-api</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+      <version>3.0.1</version>
```

Comment:
@g4s8 There's a newer version of the API. ``4.0.0``, see [here](https://mvnrepository.com/artifact/javax.servlet/javax.servlet-api)